src/components/Maps/MapPage.tsx

```diff
@@ -31,13 +35,15 @@ const MapPage = ({
   const [interestPinTitle, setInterestPinTitle] = React.useState('');
   const [interestPinLocation, setInterestPinLocation] = React.useState('');
   const [mapTypeId, setMapTypeId] = React.useState(MapTypes.ROADMAP);
+  const [openSidebar, setOpenSidebar] = React.useState(false);
   const defaultMap = {
     zoom: 11,
     center: {
       lat: 43.470846,
       lng: -80.538473,
     },
   };
+  const geolocationErrorCallback = undefined;
```

Comment:
we're going to define this when we tackle error handling right? If so, maybe add a TODO here so we don't forget

Reply:
Yeah I wasn't sure what our error handling policy was. I will discuss with Peter about this.